test/test_datasets.py

```diff
@@ -21,6 +21,8 @@
 import torch.nn.functional as F
 from torchvision import datasets
 
+random_state_numpy = np.random.RandomState(0)
```

Comment:
we should create a local RandomState object in each test function   instead of having a global one in each file. Otherwise, tests are still dependent on each other withing a single module.

Also, no strong opinion on that but `np_rng` would be shorter and still be a descriptive name, so I'd suggest to use that instead